openpype/tools/studio_tools/app.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+from Qt import QtCore
+from Qt import QtWidgets
+from openpype import style
+from openpype.settings import get_project_settings
+from openpype.tools.utils import lib as tools_lib
+
+import scriptsmenu
+import logging
+
+log = logging.getLogger(__name__)
+module = sys.modules[__name__]
+module.window = None
+
+
+class StudioToolsDialog(QtWidgets.QWidget):
```

Comment:
It is a question if `StudioToolsDialog` should be a widget. Wouldn't be better to just popup menu at current mouse position?